data/common.yaml

```diff
@@ -33,4 +43,64 @@ profile::network_vlan::vlans:
   '30':
     vlan_name: Default-VLAN30
     shutdown: false
-    ensure: present
\ No newline at end of file
+    ensure: present
+
+# manage interfaces
+profile::interface:interfaces:
+  'Loopback1':
+    enable: true
+    description: 'Loopback Interface'
+  'GigabitEthernet1/0/1':
+    enable: true
+    description: '101'
+    speed: '100m'
+  'GigabitEthernet1/0/2':
+    enable: true
+    description: '102'
+    speed: '1g'
+  'GigabitEthernet1/0/3':
+    enable: true
+    description: '103'
+    speed: '10m'
+  'GigabitEthernet1/0/4':
+    enable: true
+    description: '104'
+    speed: '1g'
+
+# manage IOS items through commands
+profile::commands:
```

Comment:
@murdok5 This should be `profile::commands::commands` to use automatic parameter loading.